CHANGES.md

```diff
@@ -6,7 +6,28 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
-## Trino Python client 0.316.0
+## Release 0.317.0
+
+* Add support for creating tables containing `JSON` columns and reading and
+  writing to them with SQLAlchemy.
+  ([#194](https://trinodb/trino-python-client/issues/194))
+* Properly propagate query failures to the client when using `fetchone`.
+  ([#95](https://trinodb/trino-python-client/issues/95))
+* Fix queries returning a single row from sometimes appearing as failed on the
+  server. ([#220](https://trinodb/trino-python-client/issues/220))
+* Fix query failures when using SQLAlchemy `Table` without a catalog name
+  provided. ([#237](https://trinodb/trino-python-client/issues/237))
```

Comment:
@mdesmet Does this correctly capture the user-observed bug underlying #237 (fixed in #240)?

Reply:
ping @mdesmet